vpr/src/util/vpr_utils.cpp

```diff
@@ -858,13 +858,13 @@ t_pb_graph_pin* get_pb_graph_node_pin_from_model_port_pin(const t_model_ports* m
 }
 
 //Retrieves the atom pin associated with a specific CLB and pb_graph_pin
+// Warning: Not all pb_graph_pins are associated with an atom pin!
```

Comment:
I'd expand on this. Some represents nodes/pins within the pb_graph hierarchy, but not on a primitive.